projects/jdwp/defs/schema.py

```diff
@@ -32,10 +28,37 @@ class PrimitiveType(Enum):
     LOCATION = "location"
 
 
+class IntegralType(Enum):
+    """Integral type class."""
+
+    STRING = "string"
+    INT = "int"
+    BYTE = "byte"
+    BOOLEAN = "boolean"
+
+
+@dataclass(frozen=True)
+class ArrayLength:
+    """Array length class."""
+
+    type: IntegralType
+
+
+class Field:
```

Comment:
Add `from __future__ import annotations` at the top instead of forward declaring Field: https://peps.python.org/pep-0563/